### PR TITLE
Notifier added to show errors related to backend middleware

### PR DESCRIPTION
**Pull request by Dzeranov**

### CODE REVIEW

File: src/index.jsx [hunk 1]
```diff
@@ -35,6 +36,7 @@ async function init() {
       store.dispatch(setUserDetails(user));
       store.dispatch(signIn());
     } catch (err) {
+      notifier.error(err.message);
```
Comment (vkomodey) [inline src/index.jsx:39]: What is error message will be shown to the user in case of an expired or non-existing token here?

Comment (vkomodey) [reply src/index.jsx:39]: ping @Dzeranov

Comment (Dzeranov) [reply src/index.jsx:39]: Hm, it will work not as expected.

Comment (vkomodey) [reply src/index.jsx:39]: what do you mean?

Comment (vkomodey) [reply src/index.jsx:39]: ping @Dzeranov

Comment (Dzeranov) [reply src/index.jsx:39]: It will show 'Please authenticate'.

Comment (vkomodey) [reply src/index.jsx:39]: kk

**Pull request merged**